queue/src/fiskaltrust.Middleware.Storage.AzureTableStorage/DatabaseMigrator.cs

```diff
@@ -17,13 +19,16 @@ public class DatabaseMigrator
         private readonly TableServiceClient _tableServiceClient;
         private readonly QueueConfiguration _queueConfiguration;
 
+        public bool? MigratedFrom1_2 { get; set; }
+
         private readonly IAzureTableStorageMigration[] _migrations;
 
         public DatabaseMigrator(ILogger<IMiddlewareBootstrapper> logger, TableServiceClient tableServiceClient, BlobServiceClient blobServiceClient, QueueConfiguration queueConfiguration)
         {
             _logger = logger;
             _tableServiceClient = tableServiceClient;
             _queueConfiguration = queueConfiguration;
+            MigratedFrom1_2 = null;
```

Comment:
Do we still need that flag after moving to queuerow based partitionkeys?

Reply:
No, but I'd like to keep it so we have the info should we need it 😁